cli/rack/__init__.py

```diff
@@ -501,7 +500,7 @@ def build_manifest_driver(
     zipfile_path: Path
 ) -> None:
 
-    with tempfile.TemporaryDirectory() as outdir:
+    with TemporaryDirectory() as outdir:
```

Comment:
Is it the case that `builder.model_graphs` and `builder.data_graphs` depend on having created the archive?
At face value it looks like it's never explicitly used, but I assume there's a reason why it needs to be created, even if temporarily.

Reply:
I'm currently only collecting those graph names as debugging output. I am imagining that we could extend that in the future to report and error if the URLs were missing from the manifest's footprint: